### ISSUE TITLE
[NetEdit] Crash with the precision.geo option

### ISSUE THREAD

**Issue by bcoueraud87** [eclipse-sumo/sumo#13881]
If you start NetEdit without any input files, and set the `precision.geo` option right from the start, then NetEdit crashes upon clicking OK.

**Comment by m-kro** [comment]
It crashes when editing other options, too, as long as no network is loaded.

**Comment by bcoueraud87** [comment]
> It crashes when editing other options, too, as long as no network is loaded.
Yes, but I found also options for which it doesn't crash too, don't remember them though.

**Comment by palvarezlopez** [comment]
Cannot be reproduced. Probably fixed during other ticket.
